README.rst

```diff
@@ -15,6 +15,22 @@ reserved.
 
 See the end of this file for further copyright and license information.
 
+General Information
+-------------------
+
+- Website: https://www.python.org
+- Source code: https://github.com/python/cpython
+- Issue tracker: https://bugs.python.org
+- Documentation: https://docs.python.org
+- Developer's Guide: https://cpython-devguide.readthedocs.io
+
```

Comment:
Thanks, @ned-deily :)
I changed the url on the `master` branch in GH-295. Once that is merged, I'll cherry-pick that to this PR. Will that work?